src/test/java/org/gogame/server/repositories/GameJournalRepositoryIntegrationTests.java

```diff
@@ -0,0 +1,82 @@
+package org.gogame.server.repositories;
+
+import org.gogame.server.domain.entities.GameActionType;
+import org.gogame.server.domain.entities.GameJournalEntity;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class GameJournalRepositoryIntegrationTests {
+
+    @Autowired
+    private final GameRepository gameRepo;
+
+    @Autowired
+    private final UserRepository userRepo;
+
+    @Autowired
+    private final GameJournalRepository gameJournalRepo;
+    @Autowired
+    public GameJournalRepositoryIntegrationTests(GameRepository gameRepo,
```

Comment:
I'm pretty sure you don't need `@Autowired` on fields if it's on the constructor

Reply:
Fixed everywhere, thanks